src/main/java/org/fundacionjala/sfdc/commons/DriverManager.java

```diff
@@ -13,12 +13,14 @@
     private static DriverManager driverManager;
     private static WebDriver driver;
     private static WebDriverWait wait;
-    private static final int SECONDS_WAIT = 90;
+    private static final int SECONDS_WAIT = 360;
 
     /**
      * Constructor, private to apply singleton pattern.
      */
     private DriverManager() {
+        System.setProperty("webdriver.gecko.driver", "geckodriver.exe");
+        driver = new FirefoxDriver();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this assignment of "driver". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3010)